factory/processComponents.go

```diff
@@ -715,7 +711,7 @@ func (pcf *processComponentsFactory) indexGenesisAccounts() error {
 		genesisAccounts = append(genesisAccounts, userAccount)
 	}
 
-	pcf.outportHandler.SaveAccounts(genesisAccounts)
+	pcf.statusComponents.OutportHandler().SaveAccounts(uint64(pcf.coreData.GenesisNodesSetup().GetStartTime()), genesisAccounts)
```

Comment:
please refresh my memory: why do we have a time parameter passed to the SaveAccounts method?

Reply:
In order to index on the `elasticsearch` database information about accounts history, we need the timestamp of the block ( in this case we need the timestamp of the genesis block)

Reply:
ok